.env.sample

```diff
@@ -0,0 +1,15 @@
+# .env.local
+
+# Select an experiment collection
+# Prod link
+RECORDS_URL="https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/nimbus-desktop-experiments/records"
+
+# Preview link
+# RECORDS_URL="https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/nimbus-preview/records"
+
+# Secrets
+EXPERIMENTER_API_KEY=
+NODE_ENV="development"
```

Comment:
Let's wait on adding the EXPERIMENTER_API_KEY, as I'm not sure that's exactly how OAuth Connect works, and we don't need it yet anyway.

Maybe change "Secrets" to "Misc" for now?